components/Chat/MessageRow.tsx

```diff
@@ -9,7 +9,7 @@ interface Props {
 const MessageRow = ({className, icon, children}: Props) => {
   return (
     <div className={className} style={{overflowWrap: "anywhere"}}>
-      <div className="relative m-auto flex max-w-2xl gap-2 p-4 px-0 py-4 text-base">
+      <div className="relative m-auto flex sm:max-w-screen-sm md:max-w-screen-md lg:max-w-screen-lg gap-2 p-4 px-0 py-4 text-base">
```

Comment:
This only makes the code block wider. It should resize with the window. Otherwise the right sidebar, or the expand button for the side bar, disappears too often.